Dockerfile-hydroshare.template

```diff
@@ -1,10 +0,0 @@
-
```

Comment:
Are all of these Dockerfile deletions intentional? Looks like a bad merge conflict resolution.